go-controller/pkg/testing/exec.go

```diff
@@ -165,7 +198,12 @@ func (f *FakeExec) Command(cmd string, args ...string) kexec.Cmd {
 	// expected command list, or if the expected command list has been
 	// completely used and we are executing more commands
 	if expected == nil {
-		klog.Fatalf("Unexpected command: %s\n\n%s", executed, f.internalErrorDesc())
+		f.receivedUnexpected = true
+		f.unexpectedCommand = executed
+		klog.Warningf("Unexpected command: %s\n\n%s", executed, f.internalErrorDesc())
+		debug.PrintStack()
+		// there's not point in returning the fake cmd as expected is nil
```

Comment:
`there's not point in returning the fake cmd as expected is nil` -> `there's no point in returning the fake cmd since expected is nil`